apps/studio/src/server/modules/resource/resource.service.ts

```diff
@@ -98,31 +98,42 @@ export const updatePageById = (
   page: Partial<Omit<Page, "id">> & { id: number },
 ) => {
   const { id, ...rest } = page
-  return db.transaction().execute((tx) => {
-    return tx
-      .updateTable("Resource")
-      .set(rest)
-      .where("id", "=", id)
-      .executeTakeFirstOrThrow()
-  })
+  return db
+    .updateTable("Resource")
+    .set(rest)
+    .where("id", "=", id)
+    .executeTakeFirstOrThrow()
 }
 
-export const updateBlobById = (props: {
+// NOTE: This id here is the page id
```

Comment:
might be just rename var to be explicit instead?

Reply:
shouldn't we also use siteId for the fetch?

Reply:
each blob is linked to 1 resource, which is linked to 1 site. don't really see how `siteId` would be used here - could you elaborate?

also renamed to `pageId`

Reply:
Basically suggestion is to standardise our queries - i.e. to use both siteId and pageId to query in the where clause. 

where siteId = XYZ AND resource.Id = ABC

Reply:
ok sure